mm/kfence.c

```diff
@@ -95,7 +112,7 @@ static void __meminit kfence_protect(unsigned long addr)
 
 	addr_end = addr + PAGE_SIZE;
 	pte = lookup_address(addr, &level);
-	BUG_ON(!pte);
+	kfence_assert(pte);
```

Comment:
This needs some error-recovery. I assume that this code cannot just continue, otherwise it'll try to deref pte below.

Reply:
Yes, I'll make the function return bool to track the error status.